about.html

```diff
@@ -184,37 +186,58 @@ <h2 class="mb-20"> Work For Your <span> Incredible Success </span> </h2>
                                   </button>
                                 </li>
                                 <li class="nav-item" role="presentation">
-                                    <button class="nav-link" id="abt3-tab" data-bs-toggle="pill" data-bs-target="#abt1" type="button" role="tab" aria-controls="pills-profile" aria-selected="false">
+                                    <button class="nav-link" id="abt3-tab" data-bs-toggle="pill" data-bs-target="#abt3" type="button" role="tab" aria-controls="pills-profile" aria-selected="false">
                                             Our Goal
                                     </button>
                                 </li>
                             </ul>
                             <div class="tab-content" id="pills-tabContent">
                                 <div class="tab-pane fade show active" id="abt1" role="tabpanel">
-                                    <p class="text"> The uploading and updating processes made by suppliers can be streamlined through front-end dashboards that create better ease of access. </p>
+                                 <p class="text"> 
+                                    Our Mission is to provide expert tax guidance and support to individuals 
+                                    and businesses, empowering them to navigate complex tax regulations with confidence. 
+                                     </p>
                                     <div class="d-flex align-items-center mt-40">
                                         <div class="btns">
                                             <a href="#" class="btn rounded-pill blue5-3Dbutn hover-blue2 sm-butn fw-bold">
-                                                <span> Learn More </span>
+                                                <span>Learn More</span>
                                             </a>
                                         </div>
                                         <div class="inf ms-3">
                                             <p class="color-999"> Support Email </p>
-                                            <a href="#" class="fw-bold color-000"> info@XX.com </a>
+                                            <a href="#" class="fw-bold color-000"> pleno@bigpond.com </a>
```

Comment:
This link isn't working. Try out `mailto:<email-address>`.

Source: https://css-tricks.com/snippets/html/mailto-links/